src/content/doc-sdk-golang/methods/raw-query.mdx

```diff
@@ -49,7 +49,32 @@ db.RawQuery(sql string, vars map[string]interface{})
 ### Example usage
 
 ```go
-// Assign the variable on the connection
-vars := map[string]interface{}{"table": "person"}
-result, err := db.RawQuery("CREATE person; SELECT * FROM type::table($table);", vars)
+func QueryRaw(db *DB, queries *[]QueryStmt) error {
```

Comment:
This is the definition of the existing [QueryRaw](https://pkg.go.dev/github.com/surrealdb/surrealdb.go#QueryRaw) function. It just copies the code from the library here. There should be an example using `QueryRaw` instead.